indico/modules/events/contributions/templates/display/subcontribution_display.html

```diff
@@ -10,7 +10,9 @@
 {%- endblock %}
 
 {% block subtitle -%}
-    {{ subcontrib.title }}
+    <a href="{{ url_for('contributions.display_contribution', subcontrib.contribution) }}">
```

Comment:
linking the subcontribution title to the contribution page? isn't that confusing?